js/build.js

```diff
@@ -280,11 +288,20 @@ Fliplet.Widget.instance('lock', function(data) {
           _this.focusOnElement($lock.find('.state[data-state=setup]'));
         }
       },
-      useTouchId: function() {
-        return Fliplet.User.Biometrics.verify({
-          title: 'Use your fingerprint to unlock your app',
-          description: 'Enter Passcode'
-        }).then(function() {
+      /**
+       * Ask the user to authenticate via the Biometrics
+       * @param {String} type
+       */
+      useBiometrics: function(type) {
+        var options = {
+          title: 'Use ' + getBiometricsDescription(type, true) + ' to unlock your app'
+        };
+
+        if (Modernizr.ios) {
+          options.description = 'Enter Passcode';
+        }
+
+        return Fliplet.User.Biometrics.verify(options).then(function() {
```

Comment:
This is a question for the JS API. What happens when title or description is undefined? If it creates an error, then we should add a default string in the JS API or just `""`. I'm just aware that Cordova plugins are usually very specific about what it needs.

Reply:
@tonytlwu I'll run some tests in the native app tests and confirm.